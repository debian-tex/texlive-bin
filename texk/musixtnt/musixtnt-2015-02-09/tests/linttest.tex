\input musixtex
\instrumentnumber2
\setclef1{\bass}
\setstaffs12
\startpiece\addspace\afterruleskip%
\Notes\qa J|\qa l&\qa n\en
\Notes\qa J|\qa l\en
\Notes\qa J&\qa n\en
\Notes\qa J|\qa l&\qa n&\qa q\en
\zstoppiece\end
