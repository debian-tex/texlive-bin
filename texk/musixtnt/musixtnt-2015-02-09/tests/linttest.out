
Too few instruments in notes on line 7:
\Notes\qa J|\qa l\en
Too few staffs for instrument 1 in notes on line 8:
\Notes\qa J&\qa n\en
Too many fields in notes on line 9:
\Notes\qa J|\qa l&\qa n&\qa q\en
